whiteboard/public/main.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+const WS_URL = 'ws://0.0.0.0:3000/ws'
```

Comment:
1. can't be the hardcoded hostname and port, get it from location.origin (or something similar)
2. the scheme must reflect the http(s) scheme:
  - if http -> ws
  - if https -> wss

Reply:
Done